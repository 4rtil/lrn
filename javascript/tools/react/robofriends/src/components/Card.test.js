import { shallow, mount, render } from 'enzyme';
import React from 'react';
import Card from './Card.js';

it('validates Card component against snapshot', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
})