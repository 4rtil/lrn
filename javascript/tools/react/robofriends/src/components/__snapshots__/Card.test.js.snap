// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validates Card component against snapshot 1`] = `ShallowWrapper {}`;
